exercise/static/exercise/results_staff.js

```diff
@@ -1148,16 +1159,23 @@
             // TODO: Get the link to students in a proper way
             const participantsLink = $('li.menu-participants').find('a').attr('href');
 
+            // LastName and FirstName are required by Sisu export, single Name field
+            // was used earlier. Perhaps it could be removed?
             let columns = [
                 {data: "UserID", title: "UserID", class: "always-hidden col-0", type: "num", searchable: false},
                 {data: "Email", title: "Email", class: "always-hidden col-1", type: "string", searchable: true},
-                {data: "StudentID", title: _("Student ID"), type: "string", width: "6em", class: "student-id stick-on-scroll col-2" },
-                //{data: "FirstName", title: "FirstName"},
-                //{data: "LastName", title: "LastName"},
-                {data: "Name", title: _("Student name"), class: "student-name stick-on-scroll col-3", type: "html", render: function(data, type, row) { return (row['UserID'] > 0 ? '<a href="' + participantsLink + row['UserID'] + '">' + data + '</a>' : ''); } },
-                {data: "Tags", title: _("Tags"), class: "tags col-4", render: function(data) { return userTagsToHTML(data); }, type: "html" },
-                {data: "Count", title: "Count", class: "col-5", visible: false, defaultContent: 0, type: "num"},
-                {data: "Total", title: _("Total"), width: "6em", class: "points total col-6", defaultContent: 0, type: "num"}
+                {data: "LastName", title: _("Last Name"), class: "student-name stick-on-scroll col-2", type: "html", render: function(data, type, row) { return (row['UserID'] > 0 ? '<a href="' + participantsLink + row['UserID'] + '">' + data + '</a>' : ''); } },
```

Comment:
With two name cells and the addition of the organization, these header cells take up quite a lot of space (especially with three sticky cells per row), leaving little space for the remaining columns with smaller screens. Perhaps these cells shouldn't be so wide by default?
Also, since there are three sticky columns, perhaps the summary names should be three columns wide.

![image](https://user-images.githubusercontent.com/50318434/127839860-ee82dbe2-7d94-42be-82fa-fe9354f1358e.png)

Reply:
If need be, I can fix the default widths while fixing #832, but the summary name cells should probably be three columns wide and fixed in this PR

Reply:
fixed colspan to 3